yelp_package/Makefile

```diff
@@ -41,13 +51,20 @@ build_trusty_docker:
 	cd dockerfiles/trusty/ && docker build -t "paastatools_trusty_container" .
 package_trusty: build_trusty_docker
 	cp ../requirements.txt ../requirements.txt.old
-	$(ADD_MISSING_DEPS)
+	cp ../debian/changelog ../debian/changelog.old
+	$(ADD_MISSING_DEPS_MAYBE)
+	$(ADD_YELP_PREFIX_MAYBE)
 	$(DOCKER_RUN_TRUSTY) /bin/bash -c "dpkg-buildpackage -d && mv ../*.deb dist/"
 	mv ../requirements.txt.old ../requirements.txt
+	mv ../debian/changelog.old ../debian/changelog
 	$(DOCKER_RUN_TRUSTY) chown -R $(UID):$(GID) /work
-itest_trusty: package_trusty
+itest_trusty: package_trusty bintray.json
 	$(DOCKER_RUN_TRUSTY) /work/yelp_package/itest/ubuntu.sh
 
+DATE := $(shell date +'%Y-%m-%d')
+PAASTAVERSION := $(shell sed 's/.*(\(.*\)).*/\1/;q' ../debian/changelog)
```

Comment:
Use `dpkg-parsechangelog` to get this. Newer versions allow you to do `dpkg-parsechangelog -l../debian/changelog --show-field Version`. In older versions, you can get by with something like `dpkg-parsechangelog -l../debian/changelog | sed -n 's/^Version: //p'`